dask/sizeof.py

```diff
@@ -54,6 +52,13 @@ def sizeof_python_collection(seq):
         return getsizeof(seq) + sum(map(sizeof, seq))
 
 
+@sizeof.register(set)
+@sizeof.register(frozenset)
+def sizeof_python_sets(seq):
+    # As of Python v3.9, it is deprecated to call random.sample() on sets
+    return sizeof_python_list_and_tuple(list(seq))
```

Comment:
Does calling `list(seq)` here create copies of items in the set `seq`? I'm wondering about any performance impacts when calling `list(...)` on a large `set`

Reply:
Generally no, `list(seq)` will copy each item by reference for everything beside basic Python types like `int` or `bool`